hack/templates/resources/90-shoot.yaml.tpl

```diff
@@ -219,6 +219,8 @@ spec:
     % endif
     % if cloud == "local":
     local:
+      machineImage:
```

Comment:
Can we remove this now that we have defaulting in the admission?

Reply:
I think it would be interesting to have it there as an example, even if it is not strictly necessary, Maybe as a comment:
```
# specify the operating system image for local environment
# machineImage:
#   name: coreos
```

Reply:
OK, agreed. Let's make it a comment